### DISCUSSION TITLE
vscode CTRL+Click go to component file instead of component.d.ts ?

### DISCUSSION THREAD

**Discussion by zecka** [nuxt/nuxt#16270] [Questions] [answered]
How to configure vscode to resolve component path ? 

Actually when click on CTRL + Click on a component inside a template the .nuxt/components.d.ts file is open instead of component file

![Kapture 2021-10-17 at 16 44 10](https://user-images.githubusercontent.com/18116930/137632560-45875dfc-7aa0-4326-a8e2-3f5af96456e0.gif)

**Comment by silvanarmbruster** [comment]
Could you manage to solve it? I'm having the same issue with my vscode 😄

**Comment by TheAlexLichter** [comment, marked as answer]
## Update: This is built-in in the latest Vue.js VS Code extension

---

### Old 

You can use https://github.com/antfu/vscode-goto-alias to get automatically redirected ☺️

**Reply by zecka** [reply]
I tried with this extension but I am still not redirected to the component with ctrl+click  😔

Note that I also enable the Takeover mode of Volar as it [suggested in extension description](https://github.com/antfu/vscode-goto-alias#working-with-vue-components)

![Kapture 2023-04-01 at 15 47 47](https://user-images.githubusercontent.com/18116930/229292935-94bd7aeb-29e8-43ca-b6f3-4505c3f7a142.gif)

**Reply by TheAlexLichter** [reply]
Follow up in the linked repo and this PR please - https://github.com/antfu/vscode-goto-alias/pull/7

**Comment by husayt** [comment]
I can't make it with nuxt 4. And there in no takeover mode anymore. What is the latest suggested approach?


thanks in advance

**Reply by yhagus** [reply]
Same here, got the same issue in WebStorm

**Comment by alexanderhorner** [comment]
Please reopen this @TheAlexLichter

**Reply by TheAlexLichter** [reply]
Why? This is even built into Vue's VS Code extension now 🤔

**Reply by alexanderhorner** [reply]
> Why? This is even built into Vue's VS Code extension now 🤔

It doesn't work for me anymore. I saw others have the same issue in this thread and assumed it was an issue that needs to be fixed by Nuxt.

**Reply by bogdan-tykhonov** [reply]
@TheAlexLichter Same issue for me, doesn't work with the latest version of Vue's VS Code extension

**Reply by ranghaus** [reply]
> @TheAlexLichter Same issue for me, doesn't work with the latest version of Vue's VS Code extension

Dito. I'm on VSCode 1.105.1 running Nuxt 4.2.1 (with Nitro 2.12.9, Vite 7.2.4 and Vue 3.5.24)

**Reply by lilkimo** [reply]
not working, nuxt 4.1.2 & vuejs/language-tools/vscode 3.1.5

**Reply by TheAlexLichter** [reply]
Will be re-released in the next Vue Extension version!
See https://github.com/vuejs/language-tools/pull/5733#issuecomment-3581022372
